### DISCUSSION TITLE
QInput :value is missing in V2

### DISCUSSION THREAD

**Discussion by a-salimkhanov** [quasarframework/quasar#12267] [Ideas / Proposals] [answered]
We could use something like this in Quasar V1
```
<q-input
  label="Todo"
  :value="'Buy milk'"
  readonly />
```

But in Quasar V2 we don't have `value` option for `q-input`.

Why someone would need that if we have `v-model`?!

Because you can't do something like this with `v-model`:

```
<q-input :value="user.name ? `User is ${user.name}` : `We don't know user name`" readonly />
```

Thanks in advance

**Comment by yusufkandemir** [comment, marked as answer]
Hi, `value` is now `model-value` because of Vue 3 changes. Here are some references to learn about it:

https://quasar.dev/start/upgrade-guide#vue-3-breaking-changes-examples
https://v3.vuejs.org/guide/migration/v-model.html

https://quasar.dev/vue-components/input#qinput-api
![image](https://user-images.githubusercontent.com/6266078/151754932-8face953-500a-45e7-9008-1ac8fb7e02ba.png)

**Comment by a-salimkhanov** [comment]
I tried with `model-value` and it works.
Thanks :+1:
